conversion.md

```diff
@@ -0,0 +1,98 @@
+For Linux and Solaris based systems, this MAY involve resolution through NSS or parsing `/etc/passwd` from the extracted container's root filesystem to determine the values of `process.user.uid` and `process.user.gid`.
+
+In addition, a converter SHOULD set the value of `process.user.additionalGids` to a value corresponding to the user described by `Config.User`.
+For Linux and Solaris based systems, this MAY involve resolution through NSS or parsing `/etc/group` and determining the group memberships of the user specified in `process.user.uid`.
+If the value of [`user`](config.md#properties) in `Config.User` is numeric, the converter SHOULD NOT modify `process.user.additionalGids`.
+
+If `Config.User` is not defined, the converted `process.user` value is implementation-defined.
+If `Config.User` does not correspond to a user in the container's context, the converter MUST return an error.
+
+## Optional Fields
+
+Certain image configuration fields are not applicable to all conversion usecases, and thus are optional for configuration converter to implement.
+A compliant configuration converter SHOULD provide a way for users to extract these fields into the generated runtime configuration:
+
+| Image Field           | Runtime Field      | Notes |
+| --------------------- | ------------------ | ----- |
+| `Config.Memory`       | platform dependent | 1     |
+| `Config.MemorySwap`   | platform dependent | 1     |
+| `Config.CPUShares`    | platform dependent | 1     |
+| `Config.ExposedPorts` | none               | 2     |
+| `Config.Volumes`      | `mounts`           | 3     |
+
+1. The meaning of these fields is platform dependent, and converters MAY ignore these fields.
+   On Linux these image fields correspond to the runtime fields `linux.resources.memory.limit`, `linux.resources.memory.swap`, and `linux.resources.cpu.shares` respectively.
+2. The runtime configuration does not have a corresponding field for this image field.
+   Converters SHOULD set [`org.opencontainers.imageSpec.exposedPorts`] in `annotations` to a comma-separated concatenation of the keys given in `Config.ExposedPorts`, unless doing so will [cause a conflict](#annotations).
+3. The converter MUST set the `destination` of the mountpoint to the value specified in `Config.Volumes`, the other fields are platform and context dependent.
+
+## Annotations
+
+There are three ways of annotating an OCI image in this specification:
+
+1. `Config.Labels` in the [configuration](config.md) of an image.
+2. `annotations` in the [manifest](manifest.md) of an image.
+3. `annotations` in the [manifest list](manifest-list.md) of an image.
+
+A converter SHOULD extract all three of these sets of annotations to `annotations` in the runtime configuration.
+However, since there may be conflicts between different sets of annotations (same key but different value), the order of precedence SHOULD be in the following (ascending) order:
```

Comment:
> To tell you the truth, this should probably be the opposite order. Manifests and Manifest-List tend not to survive to the runtime portion and are more oriented towards distribution.

These are in order of ascending precedence, and while that's a somewhat unusual way to list precedence, it does mean that `Config.Labels` has the highest precedence.  I think that's what you're asking for too.